storage/timecache/export_test.go

```diff
@@ -0,0 +1,26 @@
+package timecache
+
+import "time"
+
+func (tc *TimeCache) Keys() []string {
+	tc.mut.Lock()
+	defer tc.mut.Unlock()
+
+	return tc.keys
+}
+
+func (tc *TimeCache) KeyTime(key string) (time.Time, bool) {
+	tc.mut.Lock()
+	defer tc.mut.Unlock()
+
+	val, ok := tc.data[key]
```

Comment:
you could return directly.

Reply:
Tried, didn't work as `return tc.data[key]` tells golang to return only the value. The variant with bool output var is not used here